templates/Dockerfile-rpm.tmpl

```diff
@@ -16,6 +17,8 @@ RUN yum groupinstall -y 'Development Tools' \
         libicu-devel \
         libxml2-devel \
         libxslt-devel \
+        llvm-toolset-7-clang \
```

Comment:
Shouldn't the PostgreSQL development packages transitively depend on this stuff if supported on a given platform? I think these things became necessary with the addition of the JIT stuff, but I also was able to deal with it by just relying on the transitive dependencies for each platform where JIT was or wasn't supported, I think.

What error made this necessary?

Reply:
We need llvm for JIT, and it was missing. The builds failed when `--with-llvm` was supplied